overlay/triplets/x64-osx.cmake

```diff
@@ -0,0 +1,11 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
```

Comment:
Rename this file to `x64-osx-mixxx` to avoid confusion with the upstream triplet.